Telerik.Examples.Mvc/Telerik.Examples.Mvc/Telerik.Examples.Mvc.csproj

```diff
@@ -22,6 +22,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="7.0.6" />
     <PackageReference Include="System.Drawing.Common" Version="7.0.0" />
     <PackageReference Include="Telerik.UI.for.AspNet.Core.Trial" Version="2023.1.425" />
+    <PackageReference Include="Telerik.Web.Spreadsheet.Trial" Version="2023.3.1010" />
```

Comment:
This should be updated as follows, since the [raw binary file](https://github.com/telerik/ui-for-aspnet-core-examples/pull/76/files#diff-f803e2bfb96feb85b6ba139449c4e1e6c73c89f6f07d5ed2e7866b113df72c07) of the NuGet package references a different version:

```
<PackageReference Include="Telerik.Web.Spreadsheet.Trial" Version="2023.1.426" />
```